From: Sebastian Ramacher <sramacher@debian.org>
Date: Wed, 27 May 2020 00:13:07 +0200
Subject: cmake: relax versions for mad and libid3tag

New enough versions are available in Debian, but there versions are not
set correctly.
---
 cmake-proxies/CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/cmake-proxies/CMakeLists.txt b/cmake-proxies/CMakeLists.txt
index 9ca7286..b8d981c 100644
--- a/cmake-proxies/CMakeLists.txt
+++ b/cmake-proxies/CMakeLists.txt
@@ -138,8 +138,8 @@ addlib( portaudio-v19      portaudio   PORTAUDIO   YES   YES   "" )
 #
 #       directory          option      symbol      req   chk   version
 addlib( ffmpeg             ffmpeg      FFMPEG      NO    NO    "libavcodec >= 51.53" "libavformat >= 52.12" "libavutil >= 52.66" )
-addlib( libid3tag          id3tag      LIBID3TAG   NO    YES   "id3tag >= 0.15.1b" )
-addlib( libmad             mad         LIBMAD      NO    YES   "mad >= 0.15.1b" )
+addlib( libid3tag          id3tag      LIBID3TAG   NO    YES   "id3tag >= 0.15.0" )
+addlib( libmad             mad         LIBMAD      NO    YES   "mad >= 0.15.0" )
 addlib( libnyquist         nyquist     NYQUIST     NO    YES   "" )
 addlib( libvamp            vamp        VAMP        NO    YES   "vamp >= 2.5" "vamp-hostsdk >= 2.5" )
 addlib( libogg             ogg         LIBOGG      NO    YES   "ogg >= 1.3.1" )
